app/src/components/Day.js

```diff
@@ -5,13 +5,17 @@ import Button from "react-bootstrap/Button";
 import Card from "react-bootstrap/Card";
 import ListGroup from "react-bootstrap/ListGroup";
 import ListGroupItem from "react-bootstrap/ListGroupItem";
+import DatePicker from "react-datepicker";
+import { Link, useParams, useHistory } from "react-router-dom";
 
 import { moonPhases } from "../MoonPhases";
 import * as apiClient from "../apiClient";
 //rename data to props
 const Day = ({ initialData }) => {
-  const today = dayjs();
-  const [date, setDate] = React.useState(today);
+  let { day } = useParams();
+  let history = useHistory();
+  //const [startDate, setStartDate] = React.useState(new Date(day));
+  const [date, setDate] = React.useState(dayjs(day));
```

Comment:
maybe this date/day section has to do with why the route is not working? Just a guess